h2o-py/tests/testdir_misc/pyunit_gh_15861_no_datatable.py

```diff
@@ -0,0 +1,54 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+from h2o.utils.shared_utils import (can_use_datatable)
+import time
+
+# here, we delete the data table and make sure the code will run.
+def test_frame_conversion(dataset, compareTime):
+    print("Performing data conversion to pandas for dataset: {0}".format(dataset))
+    h2oFrame = h2o.import_file(pyunit_utils.locate(dataset))
+     # convert frame in single thread
+    startT = time.time()
+    original_pandas_frame = h2oFrame.as_data_frame()
+    oldTime = time.time()-startT
+    print("H2O frame to Pandas frame conversion time: {0}".format(oldTime))
+    # convert frame using datatable
+    startT = time.time()
+    new_pandas_frame = h2oFrame.as_data_frame(multi_thread=True)
+    newTime = time.time()-startT
+    print("H2O frame to Pandas frame conversion time using datatable: {0}".format(newTime))
+    if compareTime: # disable for small dataset.  Multi-thread can be slower due to more overhead
```

Comment:
It seems that we don't need to calculate and compare times in this method.

Reply:
Yeah, you are right.  I shortened the test to just capture the warning message instead.  Thanks.